src/CashTransactionLine.php

```diff
@@ -0,0 +1,264 @@
+<?php
```

Comment:
Same for this class: is this (partially) copied from another class? Please see if you can move some code into traits then.

Reply:
Yes it is. I will look into it!

Reply:
@willemstuursma Here I followed the documentation of Twinfield. As far I can see, according to the documentation, each of the transactions lines (Bank, Cash, Sales) are slightly different in which value is valid/allowed for each line type. Do you have an example where we could share code?

Reply:
It's too bad they are all slightly different. Thanks for looking into it.